composer.json

```diff
@@ -44,7 +45,24 @@
 		},
 		"bamarni-bin": {
 			"target-directory": "bin/composer"
-		}
+		},
+        "changelogger": {
+			"changelog": "./changelog.txt",
+        	"formatter": {
+        		"filename": "bin/changelogger/WCAdminFormatter.php"
+        	},
+			"versioning": "wordpress",
```

Comment:
I think we can remove this and have it default to `semver`.
The wordpress versioning doesn't seem to hold the change significance in account (patch, minor, major), and it only outputs the major.minor version (ex: 2.7), but we use the `semver` format.

Reply:
Sounds good! I will make the change 👍

Reply:
Updated `semver` in https://github.com/woocommerce/woocommerce-admin/pull/6870/commits/94cf6a72eb429705aed8d8ef7ae35f38a17b17f8 and spacing issue in https://github.com/woocommerce/woocommerce-admin/pull/6870/commits/e1f4a694a2420c4fefaf861748b770d918902c67